DEB [20250421-18:04:16.190] thr=1   paramiko: starting thread (client mode): 0x3054f100
DEB [20250421-18:04:16.191] thr=1   paramiko: Local version/idstring: SSH-2.0-paramiko_2.7.2
ERR [20250421-18:04:16.192] thr=1   paramiko: Exception: Error reading SSH protocol banner[WinError 10053] 你的主机中的软件中止了一个已建立的连接。
ERR [20250421-18:04:16.193] thr=1   paramiko: Traceback (most recent call last):
ERR [20250421-18:04:16.194] thr=1   paramiko:   File "E:\python\vp_180\py_venv\lib\site-packages\paramiko\transport.py", line 2211, in _check_banner
ERR [20250421-18:04:16.194] thr=1   paramiko:     buf = self.packetizer.readline(timeout)
ERR [20250421-18:04:16.194] thr=1   paramiko:   File "E:\python\vp_180\py_venv\lib\site-packages\paramiko\packet.py", line 380, in readline
ERR [20250421-18:04:16.197] thr=1   paramiko:     buf += self._read_timeout(timeout)
ERR [20250421-18:04:16.197] thr=1   paramiko:   File "E:\python\vp_180\py_venv\lib\site-packages\paramiko\packet.py", line 607, in _read_timeout
ERR [20250421-18:04:16.197] thr=1   paramiko:     x = self.__socket.recv(128)
ERR [20250421-18:04:16.197] thr=1   paramiko: ConnectionAbortedError: [WinError 10053] 你的主机中的软件中止了一个已建立的连接。
ERR [20250421-18:04:16.198] thr=1   paramiko: 
ERR [20250421-18:04:16.198] thr=1   paramiko: During handling of the above exception, another exception occurred:
ERR [20250421-18:04:16.198] thr=1   paramiko: 
ERR [20250421-18:04:16.198] thr=1   paramiko: Traceback (most recent call last):
ERR [20250421-18:04:16.198] thr=1   paramiko:   File "E:\python\vp_180\py_venv\lib\site-packages\paramiko\transport.py", line 2039, in run
ERR [20250421-18:04:16.199] thr=1   paramiko:     self._check_banner()
ERR [20250421-18:04:16.199] thr=1   paramiko:   File "E:\python\vp_180\py_venv\lib\site-packages\paramiko\transport.py", line 2215, in _check_banner
ERR [20250421-18:04:16.199] thr=1   paramiko:     raise SSHException(
ERR [20250421-18:04:16.200] thr=1   paramiko: paramiko.ssh_exception.SSHException: Error reading SSH protocol banner[WinError 10053] 你的主机中的软件中止了一个已建立的连接。
ERR [20250421-18:04:16.200] thr=1   paramiko: 
